test/TestAsset/ConfigPostProcessor/NonAliasedServiceConfiguration.php.out

```diff
@@ -8,7 +8,18 @@ return [
         'aliases' => [
             'foo' => 'Laminas\Form\Factory',
             'Zend\Cache\Storage\StorageInterface' => 'Laminas\Cache\Storage\StorageInterface',
+            'Zend\Expressive\Router\RouterInterface' => 'MyService',
+            'Zend\Expressive\OtherInterface' => 'OtherService',
+            'OtherService' => 'Mezzio\OtherInterface',
+            'MyAlias' => 'Mezzio\AliasChain',
+            'Mezzio\Router\RouterInterface' => 'Zend\Expressive\Router\RouterInterface',
+            'Mezzio\AliasInterface' => 'Zend\Expressive\AliasInterface',
+            'Mezzio\AliasChain' => 'Zend\Expressive\AliasChain',
             'Zend\Form\Factory' => 'Laminas\Form\Factory',
         ],
+        'invokables' => [
+            'Zend\Expressive\AliasInterface' => 'MyNamespace\InvokableClass',
+            'Zend\Expressive\AliasChain' => 'MyNamespace\OtherInvokableClass',
+        ],
```

Comment:
These are not my expectations at all!

I'd expect:

```php
'aliases' => [
    'foo' => 'Laminas\Form\Factory',
    'Zend\Cache\Storage\StorageInterface' => 'Laminas\Cache\Storage\StorageInterface',
    'Zend\Expressive\Router\RouterInterface' => 'Mezzio\Router\RouterInterface',
    'Zend\Expressive\OtherInterface' => 'OtherService',
    'OtherService' => 'Mezzio\OtherInterface',
    'MyAlias' => 'Mezzio\AliasChain',
    'Mezzio\Router\RouterInterface' => 'MyService',
    'Zend\Expressive\AliasInterface' => 'Mezzio\AliasInterface',
    'Zend\Expressive\AliasChain' => 'Mezzio\AliasChain',
    'Zend\Form\Factory' => 'Laminas\Form\Factory',
],
'invokables' => [
    'Mezzio\AliasInterface' => 'MyNamespace\InvokableClass',
    'Mezzio\AliasChain' => 'MyNamespace\OtherInvokableClass',
],
```

What to note about my suggestions:

- Instead of pointing the _new_ implementations at the _old_ implementations, I switch those around (c.f. the entry for `RouterInterface`).
- I take the alias part of the `invokables` configuration and create aliases of the old names to the new names, and then replace the alias in the `invokables` with the new alias (c.f. the entry for `AliasInterface`).

I was uncertain about the `Zend\Expressive\OtherInterface` item, but realized that since it resolves to a service that uses the new name, it works.

The changes I propose will allow aliases to resolve more quickly.